src/components/MapMopsView.js

```diff
@@ -107,7 +107,7 @@ export default class MapMopsView extends Component {
       >
         <Text h4 numberOfLines={2}>{marker.title}</Text>
         <Text numberOfLines={1}>Kierunek: {marker.direction}</Text>
-        {FACILITIES.getFacilitiesIcons(marker.facilities_short)}
+        {FACILITIES.getFacilitiesIconsShort(marker.facilities_short)}
```

Comment:
This code is not working :/ You need to remove line 169: style={m()}.
Sorry, forgot about it.